### ISSUE TITLE
Add attributes to items in the media library

### ISSUE THREAD

**Issue by visionarygleam** [merelinguist/playhouse-press#6]
Photos and videos should have alt text, a caption, and information on size and filesize.

**Comment by merelinguist** [comment]
Fixed 👍
